[2025-04-04T08:09:15.351+0000] {processor.py:186} INFO - Started process (PID=37172) to work on /opt/airflow/dags/beer_data_processing_dag.py
[2025-04-04T08:09:15.416+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/beer_data_processing_dag.py for tasks to queue
[2025-04-04T08:09:15.480+0000] {logging_mixin.py:190} INFO - [2025-04-04T08:09:15.460+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/beer_data_processing_dag.py
[2025-04-04T08:09:18.702+0000] {logging_mixin.py:190} INFO - [2025-04-04T08:09:18.371+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/beer_data_processing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/beer_data_processing_dag.py", line 6, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-04T08:09:18.752+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/beer_data_processing_dag.py
[2025-04-04T08:09:19.063+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/beer_data_processing_dag.py took 4.012 seconds
[2025-04-04T08:10:32.154+0000] {processor.py:186} INFO - Started process (PID=37473) to work on /opt/airflow/dags/beer_data_processing_dag.py
[2025-04-04T08:10:32.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/beer_data_processing_dag.py for tasks to queue
[2025-04-04T08:10:32.160+0000] {logging_mixin.py:190} INFO - [2025-04-04T08:10:32.160+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/beer_data_processing_dag.py
[2025-04-04T08:10:32.294+0000] {logging_mixin.py:190} INFO - [2025-04-04T08:10:32.258+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/beer_data_processing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/beer_data_processing_dag.py", line 6, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-04T08:10:32.303+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/beer_data_processing_dag.py
[2025-04-04T08:10:32.449+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/beer_data_processing_dag.py took 0.306 seconds
